ontology/base_slims.py

```diff
@@ -153,7 +153,7 @@
         'OBI:0000845': 'RNA structure',
         'OBI:0002082': 'Reporter assay',  # 'Reporter gene assay'
         'OBI:0002675': 'Massively parallel reporter assay',
-        'NTR:0000520': 'CRISPR screen',
+        'OBI:0003659': 'CRISPR screen',
```

Comment:
do you also need to remove this term NTR:0000520 in ntr_terms.py?

Reply:
Yes, thank you for catching.